src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace GeneralizedHarmonic::BoundaryConditions {
+namespace detail {
+// @{
+/*!
+ * \brief Computes the expression needed to set bnudary conditions on Dt<VPsi>
+ *
+ * \details Computes the correction to the characteristic projected
+ * time-derivatives of the fundamental variables corresponding to \f$u^\psi\f$:
+ *
+ * \f{align}
+ * \partial_t u^{\psi}_{ab} = \partial_t u^{\Psi}_{ab} + v_{\psi} n^i C_{iab}
```

Comment:
Maybe use `->` instead of `=` here? Did you intend `\Psi` to be capitalized on RHS in first term?

Reply:
changed. Since I've now changed the function to return only the correction term on the RHS, this equation has no more lhs/rhs repetition.